.github/workflows/Dockerfile.vlm

```diff
@@ -81,7 +81,7 @@ ENV EVM_BRANCH=${EVM_BRANCH}
 
 RUN echo "EVM_BRANCH=${EVM_BRANCH}"
 
-ARG VLM_OP_GETH_BRANCH=ulm
```

Comment:
This branch doesn't exist anymore, so this will fail in the next CI execution. We have 2 options here:
- We can change it back and re-execute the CI before merging this
- Restore this branch in `op-geth` repo, and once you merge the VLM PR, come back here and open a PR changing this branch and the VLM branch. (I prefer this one.)